ddl/ddl_api.go

```diff
@@ -959,10 +954,6 @@ func setDefaultAndComment(ctx context.Context, col *table.Column, options []*ast
 	}
 
 	setTimestampDefaultValue(col, hasDefaultValue, setOnUpdateNow)
-	if col.Tp == mysql.TypeTimestamp && hasNotNull {
```

Comment:
Should we return error when timestamp has not null and doesn't have default value?